src/cache.py

```diff
@@ -1,13 +1,24 @@
+import pandas as pd
+
 
 class Cache:
 
     def __init__(self):
-        self.__cache = {}         
+        self.__cache = {}
+        self.persist_to_cache('currencies', ['USD', 'CAD', 'EUR', 'GBP'])
+        self.persist_to_cache('tickers', pd.read_csv('tickers.csv')['Symbol']
+                              .drop_duplicates().values.tolist())
```

Comment:
Would be good if these were loaded from text files or csv files, where users could customise accepted values.